compose/docker-compose-fixturenet-eth.yml

```diff
@@ -0,0 +1,89 @@
+version: '3.7'
+
+services:
+  fixturenet-eth-bootnode-geth:
+    hostname: fixturenet-eth-bootnode-geth
+    environment:
+      RUN_BOOTNODE: "true"
+      NETWORK_ID: "1212"
+    image: cerc/fixturenet-eth-geth:local
+    networks:
+      fixturenet-eth-net:
+        ipv4_address: 172.16.254.10
+
+  fixturenet-eth-geth-1:
+    hostname: fixturenet-eth-geth-1
+    environment:
+      NETWORK_ID: "1212"
+      ACCOUNT_PASSWORD: "secret1212"
+      ETHERBASE: "0xe6CE22afe802CAf5fF7d3845cec8c736ecc8d61F"
+      ENODE: "enode://af22c29c316ad069cf48a09a4ad5cf04a251b411e45098888d114c6dd7f489a13786620d5953738762afa13711d4ffb3b19aa5de772d8af72f851f7e9c5b164a@fixturenet-eth-bootnode-geth:30303"
+      JWT: "0x6cdcac3501046a08e186730dd8bd136cfaf0fdc1fc955f6e15ad3068c0ff2af0"
+    image: cerc/fixturenet-eth-geth:local
+    depends_on:
+      - fixturenet-eth-bootnode-geth
+    ports:
+      - "8545"
+    networks:
+      fixturenet-eth-net:
+
+  fixturenet-eth-geth-2:
+    hostname: fixturenet-eth-geth-2
+    environment:
+      NETWORK_ID: "1212"
+      ACCOUNT_PASSWORD: "secret1212"
+      ETHERBASE: "0xe6CE22afe802CAf5fF7d3845cec8c736ecc8d61F"
+      ENODE: "enode://af22c29c316ad069cf48a09a4ad5cf04a251b411e45098888d114c6dd7f489a13786620d5953738762afa13711d4ffb3b19aa5de772d8af72f851f7e9c5b164a@fixturenet-eth-bootnode-geth:30303"
+      JWT: "0x6cdcac3501046a08e186730dd8bd136cfaf0fdc1fc955f6e15ad3068c0ff2af0"
+    image: cerc/fixturenet-eth-geth:local
+    depends_on:
+      - fixturenet-eth-bootnode-geth
+    networks:
+      fixturenet-eth-net:
+
+  fixturenet-eth-bootnode-lighthouse:
+    hostname: fixturenet-eth-bootnode-lighthouse
+    environment:
+      RUN_BOOTNODE: "true"
+    image: cerc/fixturenet-eth-lighthouse:local
+    networks:
+      fixturenet-eth-net:
+        ipv4_address: 172.16.254.11
+
+  fixturenet-eth-lighthouse-1:
+    hostname: fixturenet-eth-lighthouse-1
+    environment:
+      NODE_NUMBER: "1"
+      JWT: "0x6cdcac3501046a08e186730dd8bd136cfaf0fdc1fc955f6e15ad3068c0ff2af0"
+      ETH1_ENDPOINT: "http://fixturenet-eth-geth-1:8545"
+      EXECUTION_ENDPOINT: "http://fixturenet-eth-geth-1:8551"
+    image: cerc/fixturenet-eth-lighthouse:local
+    depends_on:
+      - fixturenet-eth-bootnode-lighthouse
+      - fixturenet-eth-geth-1
+    ports:
+      - "8001"
+    networks:
+      fixturenet-eth-net:
+  
+  fixturenet-eth-lighthouse-2:
+    hostname: fixturenet-eth-lighthouse-2
+    environment:
+      NODE_NUMBER: "2"
+      JWT: "0x6cdcac3501046a08e186730dd8bd136cfaf0fdc1fc955f6e15ad3068c0ff2af0"
+      ETH1_ENDPOINT: "http://fixturenet-eth-geth-2:8545"
+      EXECUTION_ENDPOINT: "http://fixturenet-eth-geth-2:8551"
+      LIGHTHOUSE_GENESIS_STATE_URL: "http://fixturenet-eth-lighthouse-1:8001/eth/v2/debug/beacon/states/0"
+    image: cerc/fixturenet-eth-lighthouse:local
+    depends_on:
+      - fixturenet-eth-bootnode-lighthouse
+      - fixturenet-eth-geth-2
+    networks:
+      fixturenet-eth-net:
+
+networks:
```

Comment:
Add a comment about why we need this network defined.

Reply:
Done